### DISCUSSION TITLE
Multiple databases, multiple schemas

### DISCUSSION THREAD

**Discussion by rabet** [geldata/gel#4471] [General]
Hi guys,

is possible to have multiple `.esdl` schemas for multiple databases running under one edgedb instance? I need to have separate schemas to migrate on different DBs (for example auth.esdl -> auth db) for microservice architecture. Is this possible?

Thanks a lot!

**Comment by Sikarii** [comment]
You could give the `--database` connection flag a go along with `--schema-dir` on the migrations.
This would mean that each service needs to have their own directory (i.e. `dbschema/service1`, `dbschema/service2`).

I'm not sure how practical this is, and I've never tried something like this, so might or might not work for you.

**Comment by rabet** [comment]
Thanks a lot @Sikarii seems to be working like you've suggested 👍
